websockets/py36/protocol.py

```diff
@@ -0,0 +1,3 @@
+async def __aiter__(self):
+    while True:
+        yield await self.recv()
```

Comment:
Or to put it differently: we need to think about the pattern for handling connection termination, in order to avoid boilerplate, such as the need to wrap every `for message in websockets` in a `try/catch` clause.